tasks/managed/push-rpm-to-koji/push-rpm-to-koji.yaml

```diff
@@ -0,0 +1,155 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: push-rpm-to-koji
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: push-koji
+spec:
+  params:
+    - name: snapshotPath
+      type: string
+      description: |
+        Path to the JSON file of the mapped Snapshot spec in the data workspace.
+    - name: dataPath
+      type: string
+      description: |
+        Path to the JSON file of the merged data to use in the data workspace.
+    - name: pushSecret
+      type: string
+      description: |
+        The secret that is used for login koji instance.
+    - name: subdirectory
+      type: string
+      description: |
+        The pipeline specific directory.
+    - name: pipelineImage
+      type: string
+      description: |
+        The image url with koji, kinit installed for running the push-rpm-to-koji task,
+        please make sure you have such image or you build this image first.
+  steps:
+    - name: push-rpm-to-koji
+      # The pipelineImage will be fetching from get-secrets task with koji installed there
+      image: "$(params.pipelineImage)"
+      script: |
+        #!/usr/bin/env bash
+        set -eux
+
+        SNAPSHOT_SPEC_FILE="$(workspaces.data.path)/$(params.snapshotPath)"
+        if [ ! -f "${SNAPSHOT_SPEC_FILE}" ] ; then
+            echo "No valid snapshot file was provided."
+            exit 1
+        fi
+
+        DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"
+        if [ ! -f "${DATA_FILE}" ] ; then
+            echo "No data JSON was provided."
+            exit 1
+        fi
+
+        APPLICATION=$(jq -r '.application' "${SNAPSHOT_SPEC_FILE}")
+        NUM_COMPONENTS=$(jq '.components | length' "${SNAPSHOT_SPEC_FILE}")
+
+        PRINCIPLE=$(jq -r '.pushOptions.pushKeytab.principal' "${DATA_FILE}")
```

Comment:
I think you should similarly change the variable name (`s/PRINCIPLE/PRINCIPAL/`)